pandas/io/json/normalize.py

```diff
@@ -259,7 +259,8 @@ def _recursive_extract(data, path, seen_meta, level=0):
     result = DataFrame(records)
 
     if record_prefix is not None:
-        result.rename(columns=lambda x: record_prefix + x, inplace=True)
+        result.rename(columns=lambda x: "{p}{c}".format(p=record_prefix, c=x),
```

Comment:
can you change to 

``result = result.rename(...)``

we don't use ``inplace`` in library code